.github/workflows/release.yml

```diff
@@ -205,7 +205,7 @@ jobs:
           java-version: ${{ env.JAVA_VERSION }}
 
       - name: Build SDK
-        run: make build_${{ matrix.language }}_sdk VERSION=${{ inputs.version }}
+        run: make build_${{ matrix.language }}_sdk PROVIDER_VERSION=${{ inputs.version }}
```

Comment:
Are you sure here? Makefile seems to reference VERSION:

```
VERSION ?= 0.0.1
```

Reply:
Yup: https://github.com/pulumi/pulumi-aws-apigateway/blob/main/Makefile#L1

Reply:
It used to be VERSION but I changed it to align with what we do in the other providers.